packages/commonwealth/client/scripts/views/pages/search/helpers.tsx

```diff
@@ -151,11 +149,10 @@ const getCommunityResult = (community, setRoute) => {
 
 const getMemberResult = (addr) => {
   const profile: Profile = app.newProfiles.getProfile(addr.chain, addr.address);
-  if (addr.name) {
-    profile.initialize(addr.name, null, null, null, null, null);
-  }
 
-  if (app.isCustomDomain() && app.customDomainId() !== addr.chain) return;
+  if (app.isCustomDomain() && app.customDomainId() !== addr.chain) {
+    return null;
+  }
 
   return (
     <div key={profile.id} className="member-result-row">
```

Comment:
I am not sure if this is exactly the place that is responsible for that but clicking in the member row does nothing (but looks like it is clickable). I am redirected only when I click on the name itself. 
<img width="685" alt="image" src="https://user-images.githubusercontent.com/14819225/232074226-d3c04e33-5958-4968-b961-62c5310db881.png">